travis/build-prereqs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+# This script should build all the OPM dependencies which are installed from source.
+
+function build_dune {
+   project=$1
+   pushd . > /dev/null
+   git clone https://github.com/dune-project/${project}.git
+   cd ${project}
+   git checkout tags/v2.3.1
+   mkdir build
+   cd build
+   cmake ../
+   make
+   popd > /dev/null
+}
+
+
+function build_superlu {
+   pushd . > /dev/null
+   git clone https://github.com/starseeker/SuperLU.git
```

Comment:
> Appears to be entirely unaffiliated with the SuperLU project.

Wow - that was interesting; I have just copied the existing travis build setup for SuperLU: https://github.com/OPM/opm-core/blob/master/travis/clone-and-build-superlu.sh